lsp-mode.el

```diff
@@ -125,18 +125,6 @@
   :type 'boolean
   :package-version '(lsp-mode . "6.1"))
 
-(defcustom lsp-use-native-json t
```

Comment:
This seems to be the PR that removes this option. Do those of us on Emacs HEAD need to do anything to get native JSON parsing?

Reply:
No, it will be picked automatically. The flag was there temporary as a fallback option when we introduced native json support.

Reply:
Thank you for the very fast response! I look forward to trying `lsp-mode` in Emacs HEAD then! :smile:

Reply:
Make sure you remove the `elc` generated from the old emacs - we do some evaluations compile time depending on emacs version to make the runtime faster. (The easiest option is to delete the package and install it again).

Reply:
I wish I could see this comment earlier. Thanks!